.gitignore

```diff
@@ -10,3 +10,7 @@
 build/
 deps/
 bin/
+
+Drakhtar/Debug/Drakhtar\.log
```

Comment:
Don't add those, just nuke them, CMake doesn't generate those folders/files anymore, it's just residual stuff from your pre-cmake builds.